xbuild/src/download.rs

```diff
@@ -138,7 +138,9 @@ impl<'a> DownloadManager<'a> {
                 self.macos_sdk()?;
             }
             Platform::Android => {
-                self.rustup_target("aarch64-linux-android")?;
+                for target in self.env().target().compile_targets() {
+                    self.rustup_target(target.rust_triple()?)?;
```

Comment:
While we're at it, it looks like we should make this more generic for all platforms.

Reply:
https://github.com/rust-mobile/xbuild/pull/139